scss/globals/top/_body.scss

```
@@ -129,7 +129,10 @@ body {
 
     // 4. 条件付きスタイル（:not(.wiki-font)）
     &:not(.wiki-font) {
-        font-family: Noto Sans Japanese, sans-serif !important;
+        font-family: Noto Sans Japanese, sans-serif;
```

Stylelint (17.13.0)

[error] Expected quotes around "Noto Sans Japanese" (font-family-name-quotes)

(font-family-name-quotes)